documentation/theme/main.html

```diff
@@ -1,6 +1,32 @@
 {% extends "base.html" %}
 
-{% block announce %} 
-  The Pack SDK is currently in a limited beta.
-  <a href="https://coda.io/packsbeta">Click here</a> to learn more and apply.
-{% endblock %}
+<!-- TODO: should shape these up and should cater to specific page -->
+<meta name="description" content="Documentation for the SDK to build Coda Packs." />
+<meta
+  name="image"
+  content="https://codaio.imgix.net/docs/OZNRRq68EI/blobs/bl-O5NyMFtXuk/0a49697fee3d51730d4c66e8726be2c3deb1801e8c751f58d40e49351425eeb8a124e13cb0a2647549663df99233d80f7f92990d0e6c501ae36d40d72bd74cf558386f8204c3e64814673be573f0599673831ce7d11429c11753d276cfc758ec92cb5079?auto=format%2Ccompress&fit=max&w=1080&dpr=1"
```

Comment:
Ah this is a bit fragile, since it's a cover image from doc that could get deleted at any time. We'd want to use an image that's uploaded directly to our CDN. Do you know if it already exists there?

If it doesn't, you can upload it directly pretty easy by logging into the coda-common AWS account and going to S3 and finding the images bucket. You may need to access in #it-help for temporary access.

Reply:
ah good call ill upload

Reply:
do we garbage collect images from docs? for some reason i thought they were just around forever

Reply:
I don't actually know for sure. Presumably we do after we stop counting stuff towards your attachments doc size. But certainly we're likely to if we delete the entire doc, and I think that's the Beta doc which is now obsolete so it could go away on us.

Reply:
oh interesting but yeah makes sense that its fragile